foundry.toml

```
@@ -2,7 +2,8 @@
 src = "src"
 out = "out"
 libs = ["lib"]
-solc-version = "0.8.23"
+solc-version = "0.8.24"
```

```suggestion
solc_version = "0.8.24"
```
nit: https://book.getfoundry.sh/reference/config/solidity-compiler#solc_version